pkg/agent/flowexporter/exporter/util.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Antrea Authors
```

Comment:
`util.go` is maybe not a great name for this file. How about `certificates.go`?

Reply:
Updated. Thanks!